production.ini

```diff
@@ -104,6 +104,9 @@ use = egg:bodhi-server
 # comps_dir = /usr/share/bodhi/
 # comps_url = https://git.fedorahosted.org/comps.git
 
+## Pungi configuration files
+# pungi_modular_config_path = /etc/bodhi/pungi/fedora-modular.conf
```

Comment:
@ausil pointed out that the modular pungi config should be specific to a release. The way this config is used, there can only be one modular release in Bodhi.

Reply:
Totally true.  We'll need to fix that in post.

Reply:
I propose making this setting a template, with {release} in it, like this:

```
# pungi_modular_config_path = /etc/bodhi/pungi/{release}-modular.conf
```

Then the masher can ```.format(release=release.name)``` to determine which file it should use.

Reply:
I don't want to fix it in post (to use the music industry terminology), because once we release the setting it will be a backwards-incompatible change to fix it.

Reply:
@ausil pointed out to me that we actually need a config per repo, not per release (i.e., update and udpates-testing each need their own config file).